spec/rails_helper.rb

```diff
@@ -86,4 +86,23 @@
   end
 end
 
-Capybara.javascript_driver = :webkit
+Capybara.register_driver :headless_chrome do |app|
```

Comment:
@elbzero is working on some overall improvements to the test suite so we need to coordinate with him. If this stuff isn't needed for the bootstrap change, let's separate this out so it can be worked on independently.